src/components/bs5/video/video.data.json

```diff
@@ -4,7 +4,11 @@
     "videoId": "LDU_Txk06tM",
     "videoSize": "col-12",
     "aspectRatio": "16x9",
-    "duration": "3:12",
+    "duration": {
```

Comment:
you split the duration from h:mm or h:mm:s to the verbose way, is this what we want?

Reply:
I was viewing it from CMS point of view that the duration can have 3 separate fields of hours, minutes and seconds. 
Second thought, to avoid having to update metadata on Squiz, I'll update the helper to support existing duration string and as well as object type.